v22.1/alter-backup.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: ALTER BACKUP
-summary: Use the ALTER BACKUP statement to add `NEW_KMS` encryption keys to backups.
+summary: Use the ALTER BACKUP statement to add new KMS encryption keys to backups.
 toc: true
 docs_area: reference.sql
 ---
 
 {% include enterprise-feature.md %}
 
-<span class="version-tag">New in v22.1:</span> The `ALTER BACKUP` statement allows for new KMS encryption keys to be applied to existing encrypted backups. Each `ALTER BACKUP` statement must include the new KMS encryption key with `NEW_KMS`, and use `WITH OLD_KMS` to refer to at least one of the KMS URIs that were originally used to encrypt the backup.
+<span class="version-tag">New in v22.1:</span> The `ALTER BACKUP` statement allows for new KMS encryption keys to be applied to existing encrypted full backups. Each `ALTER BACKUP` statement must include the new KMS encryption key with `NEW_KMS`, and use `WITH OLD_KMS` to refer to at least one of the KMS URIs that were originally used to encrypt the backup.
```

Comment:
maybe we can say:

to be applied to an existing chain of encrypted backups (full + incremental)